pkg/controller/perconaservermongodb/psmdb_controller.go

```diff
@@ -446,6 +447,19 @@ func (r *ReconcilePerconaServerMongoDB) reconcileStatefulSet(arbiter bool, cr *a
 			)
 		}
 	}
+
+	sslHash, err := r.getTLSHash(cr, cr.Spec.Secrets.SSL)
+	if err != nil {
+		return nil, fmt.Errorf("get secret hash error: %v", err)
+	}
+	sfsSpec.Template.Annotations["percona.com/ssl-hash"] = sslHash
+	sslInternalHash, err := r.getTLSHash(cr, cr.Spec.Secrets.SSLInternal)
+	if err != nil && !k8serrors.IsNotFound(err) {
+		return nil, fmt.Errorf("get secret hash error: %v", err)
+	} else if !k8serrors.IsNotFound(err) {
```

Comment:
```suggestion
	} else if err == nil {
```
I think it is a better option, set annotation if hash received

Reply:
both variants are equal though `err == nil` looks more clear